MediaRecorder.html

```diff
@@ -330,14 +353,16 @@
             method is invoked, the UA MUST queue a task, using the DOM
             manipulation task source, that runs the following steps:
             <ol>
-              <li>If <code>state</code> is <code>inactive</code>, throw an
-              <code>InvalidStateError</code> DOMException and terminate these
-              steps. Otherwise:</li>
-              <li>Set <code>state</code> to <code>inactive</code> and stop
-              gathering data.</li>
-              <li>Fire an event named <code>dataavailable</code> containing the
-              Blob of data that has been gathered.</li>
-              <li>Fire an event named <code>stop</code>.</li>
+              <li>If <a>state</a> is <a href="#dom-recordingstate-inactive">
+              inactive</a>, throw an <code>InvalidStateError</code> DOMException
+              and terminate these steps. Otherwise:</li>
+              <li>Set <a>state</a> to <a href="#dom-recordingstate-inactive">
+              inactive</a> and stop gathering data.</li>
+              <li>Let <var>blob</var> be the Blob of collected data so far, then
+              <a href="#dfn-firing-a-dataavailable-event">fire a dataavailable
+              event</a> with <var>blob</var>.</li>
+              <li><a href="https://dom.spec.whatwg.org/#concept-event-fire">Fire
+              </a> an event named <code>stop</code>.</li>
```

Comment:
These last two steps are the only ones I think need to remain in the queued task.